src/main/app/controller/GetCaseController.ts

```diff
@@ -10,26 +23,26 @@ export class GetCaseController {
   //constructor(protected readonly view: string, protected readonly content: TranslationFn) {}
 
   public async getApplicantCase(req: AppRequest, res: Response): Promise<void> {
-    req.session.userCase = await GetCaseController.assignUserCase(req);
+    req.session.userCase = await GetCaseController.assignUserCase(req, undefined);
     req.session.save(() => res.redirect(APPLICANT_TASK_LIST_URL));
   }
 
   public async getRespondentCase(req: AppRequest, res: Response): Promise<void> {
-    req.session.userCase = await GetCaseController.assignUserCase(req);
+    req.session.userCase = await GetCaseController.assignUserCase(req, undefined);
```

Comment:
Why are we passing undefined?

Reply:
assign user case needs a caseid param based on that it will be different flow so